experiments/axon_geometry/README.rst

```diff
@@ -0,0 +1,49 @@
+
+How to reproduce this experiment
+--------------------------------
+
+Before reproducing this experiment, make sure that:
+
+- You have installed the ``brainlit`` package [`Documentation <https://brainlit.netlify.app/readme#installation>`_]
```

Comment:
Do you feel that we should also update the installation docs with the BigSur issues?

Reply:
@JacopoTeneggi can you remind me, does bigsur break some of our dependencies? or is it a brainlit specific issue?

@bvarjavand what would be the best way to specify that brainlit doesn't work on BigSur?

Reply:
I added item #9 [here](https://github.com/neurodata/brainlit/blob/tathey1-splinedocs/docs/install_faq.rst). @JacopoTeneggi if you think it would be better to be more specific, could you add it please?

Reply:
@tathey1 it was an issue with napari on BigSur and Python 3.9.1 https://github.com/napari/napari/issues/1393#issuecomment-745615931 I will edit the installation README